cloud/src/meta-service/txn_kv_error.h

```diff
@@ -52,7 +52,7 @@ inline const char* format_as(TxnErrorCode code) {
     case TxnErrorCode::TXN_INVALID_ARGUMENT: return "InvalidArgument";
     case TxnErrorCode::TXN_KEY_TOO_LARGE: return "Key length exceeds limit";
     case TxnErrorCode::TXN_VALUE_TOO_LARGE: return "Value length exceeds limit";
-    case TxnErrorCode::TXN_BYTES_TOO_LARGE: return "Transaction exceeds byte limit";
+    case TxnErrorCode::TXN_BYTES_TOO_LARGE: return "Too many partitions to load. Please reduce the number of partitions involved in the load.";
```

Comment:
The TXN_BYTES_TOO_LARGE error message is a low-level error that may be utilized across multiple components. It would be more appropriate to append detailed error context when this error code is caught at higher abstraction layers. For instance, when encountering TXN_BYTES_TOO_LARGE during transaction commit, we could include suggestions about partition reduction.

Reply:
I think it's fine to change this message since `TXN_BYTES_TOO_LARGE` means the transaction is too large, unless there are other reasons a transaction might be too large.